references/classification/presets.py

```diff
@@ -69,11 +69,10 @@ def __init__(
         backend="pil",
     ):
         trans = []
-
         backend = backend.lower()
         if backend == "tensor":
             trans.append(transforms.PILToTensor())
-        else:
+        elif backend != "pil":
```

Comment:
This probably still be buggy when `backend="pil"`

As it will not apply `PILtoTensor()` transform.

Reply:
Should we apply PILtoTensor when backend="pil"? I was just guided by the way it is done in the ClassificationPresetTrain class

Reply:
We probably do this in Line 83. I missed this. But it's confusing why would one apply `PILToTensor()` when backend type is `Tensor()`. Maybe I'm missing something again.

Reply:
As I see, we apply PILtoTensor at the beginning, when backend=="tensor", to do transformations over tensors, not over PIL images.

```python
trans += [
    transforms.Resize(resize_size, interpolation=interpolation, antialias=True),
    transforms.CenterCrop(crop_size),
]
```

As written in the documentation for [transforms.Resize](https://pytorch.org/vision/stable/generated/torchvision.transforms.Resize.html#torchvision.transforms.Resize):  "The output image might be different depending on its type: when downsampling, the interpolation of PIL images and tensors is slightly different, because PIL applies antialiasing. This may lead to significant differences in the performance of a network."